chatbot/model_service/builds/x86-cuda/Containerfile

```diff
@@ -1,10 +1,14 @@
 FROM quay.io/opendatahub/workbench-images:cuda-ubi9-python-3.9-20231206
 WORKDIR /locallm
-COPY builds/requirements.txt /locallm/requirements.txt
+# MODEL_FILE should be set at runtime
+ENV MODEL_FILE=llama-2-7b-chat.Q5_K_S.gguf
```

Comment:
Is this used for anything any more? I think "MODEL_PATH" might be sufficient.

Reply:
I removed both, they aren't needed for the builds